src/Cmfd.cpp

```diff
@@ -1824,6 +2104,28 @@ void Cmfd::updateBoundaryFlux(Track** tracks, FP_PRECISION* boundary_flux,
 }
 
 
+/** @brief Set a pointer to the Geometry.
+ * @param goemetry A pointer to a Geometry object.
+ */
+void Cmfd::setGeometry(Geometry* geometry){
+  _geometry = geometry;
+}
+
+
+/** @brief Set a number of k-nearest neighbor cells to use in updating
+ *         the FSR flux.
+ * @param k_nearest The number of nearest neighbor CMFD cells.
+ */
+void Cmfd::setKNearest(int k_nearest){
+
+  if (_k_nearest < 1 || k_nearest > 9)
+    log_printf(ERROR, "Unable to set CMFD k-nearest to %i. k-nearest "
```

Comment:
Change to ``%d`` to be consistent with the rest of the code, unless you think there is good reason for us to start using ``%i`` throughout.